### ISSUE TITLE
Update Argon2 Blogpost

### ISSUE THREAD

**Issue by zepatrik** [ory/web#107]
The changes from https://github.com/ory/kratos/pull/961 should be reflected there as well. Will need a new short section on concurrency.

**Comment by aeneasr** [comment]
@zepatrik is this still relevant?

**Comment by zepatrik** [comment]
Yes, the CLI changed quite a bit, and the blog post should have a whole new section about concurrent hashing. But this only has to be addressed once we release it.

**Comment by zepatrik** [comment]
blocked by https://github.com/ory/kratos/pull/1012

**Comment by vinckr** [comment]
Seems this is not blocked anymore, what needs to be changed @zepatrik ?

**Comment by zepatrik** [comment]
Right, I will put it on my list.

**Comment by aeneasr** [comment]
gentle ping @zepatrik

**Comment by zepatrik** [comment]
ah whoopsie :sweat_smile:
